bindings/web5_uniffi_wrapper/src/credentials/verifiable_presentation_1_1.rs

```diff
@@ -0,0 +1,76 @@
+use crate::{dids::bearer_did::BearerDid, errors::Result};
+use std::{sync::Arc, time::SystemTime};
+use web5::credentials::verifiable_presentation_1_1::VerifiablePresentation as InnerVerifiablePresentation;
+use web5::credentials::verifiable_presentation_1_1::VerifiablePresentationCreateOptions as InnerVerifiablePresentationCreateOptions;
+
+#[derive(Default)]
+pub struct VerifiablePresentationCreateOptions {
+    pub id: Option<String>,
+    pub context: Option<Vec<String>>,
+    pub r#type: Option<Vec<String>>,
+    pub issuance_date: Option<SystemTime>,
+    pub expiration_date: Option<SystemTime>,
+}
```

Comment:
you can actually just re-use the `web5::credentials::verifiable_presentation_1_1::VerifiablePresentationCreateOptions` here rather than re-defining the struct and mapping to/from. it's not the case with `VerifiableCredentialCreateOptions` because that struct has a property within it which has to be json-serialized over the FFI (evidence) whereas here you just have primitive types.

Reply:
I see.. yup makes sense